### ISSUE TITLE
"Copy character" button for site

### ISSUE THREAD

**Issue by foxt** [lucide-icons/lucide#926]
When using Lucide as a font (i.e. TTF), it currently only compensates for usage in CSS. It'd be nice to be able to use in software such as Figma or XD. For this it'd be nice to have a "Copy character" button, similar to the font awesome website.

**Comment by ericfennis** [comment]
Yeah that would be nice, but we can't do this one before we fix the "stable hex character" problem, see #514.

**Comment by lutzseverino** [comment]
There is no maintainable, scalable way of creating a Figma component that displays Lucide icons if neither the font provides ligatures or the website provides a "copy character" button.

I'm trying to create one right now for a project we're working on, and this is a deal breaker. To make use of the component I'm creating, I have to tell teammates to upload the font to a font inspector, copy the hexadecimal, go to Google for a website that provides the Unicode so they can copy it... a whole hassle.

So... bump!

**Comment by lutzseverino** [comment]
Will wait for a day or so, but since a bot marked this as stale, I might create a new issue.

**Comment by jguddas** [comment]
Instead of creating a duplicate issue we just have to close as duplicate, feel free to create a PR instead.

**Comment by The-Lost-Light** [comment]
Even displaying unicode would be much more convenient

**Comment by janTatesa** [comment]
Or atleast show what's the encoded code in https://unpkg.com/lucide-static@0.541.0/font/info.json

**Comment by janTatesa** [comment]
Though there is a problem that every programming language has a different syntax for unicode characters. What if there was a dropdown menu where you choose what programming language to copy the character for?
